### DISCUSSION TITLE
How to get JSON data from telebot?

### DISCUSSION THREAD

**Discussion by ALLADMINdotRU** [eternnoir/pyTelegramBotAPI#1876] [Q&A]
How to see the complete received request in json format?

**Comment by coder2020official** [comment]
Why?

**Comment by Badiboy** [comment]
Set debug level to "debug" - you will see all interaction of bot and Telegram.

**Comment by Badiboy** [comment]
You can also set simple debug as middleware to see what's passing on update received.

**Comment by ALLADMINdotRU** [comment]
Is it possible to get it somehow through the framework?

**Comment by ALLADMINdotRU** [comment]
Is it possible to get it somehow through the framework?

**Reply by Badiboy** [reply]
> Is it possible to get it somehow through the framework?

> You can also set simple debug as middleware to see what's passing on update received.

**Reply by ALLADMINdotRU** [reply]
no, I want to intercept the call and process it.
